src/Compilers/CSharp/Portable/BoundTree/BoundSpillSequence.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal partial class BoundSpillSequence
+    {
+        public BoundSpillSequence(
+            SyntaxNode syntax,
+            ImmutableArray<LocalSymbol> locals,
+            ImmutableArray<BoundExpression> sideEffects,
+            BoundExpression value,
+            TypeSymbol type,
+            bool hasErrors = false)
```

Comment:
>hasErrors [](start = 17, length = 9)

I didn't see any callers pass this parameter. Would it make sense to remove, or just inline the `hasErrors: false` into two users of this constructor?

Reply:
I'd rather leave this in its most general form. The only purpose of this method is to adapt callers who want to pass an array of expression side-effects. It is not intended to be tailored to particular clients.